auth/feature-tests/functional/step-definitions/token.steps.ts

```diff
@@ -0,0 +1,95 @@
+import 'dotenv/config'
+import { loadFeature, describeFeature } from "@amiceli/vitest-cucumber";
+import { AuthDriver, TokenExchangeResponse } from "../driver/auth.driver";
+import { AuthFixtures } from "../fixtures/auth.fixtures";
+import { expect } from "vitest";
+import { JwtDecoder } from '../helper/jwt-decoder';
+import {Then} from "cucumber";
```

Comment:
```suggestion
import { Then } from "cucumber";
```

Not your fault, but we probably want to add a formatter soon

Reply:
Reverted the spaces. Got to improve on linting rules.